pkg/controller/projectreference/projectreference_adapter_test.go

```diff
@@ -2,26 +2,43 @@ package projectreference_test
 
 import (
 	"errors"
+	"strings"
 	"time"
 
 	"github.com/golang/mock/gomock"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	clusterapi "github.com/openshift/cluster-api/pkg/util"
-	api "github.com/openshift/gcp-project-operator/pkg/apis/gcp/v1alpha1"
-	gcpv1alpha1 "github.com/openshift/gcp-project-operator/pkg/apis/gcp/v1alpha1"
-	. "github.com/openshift/gcp-project-operator/pkg/controller/projectreference"
-	mocks "github.com/openshift/gcp-project-operator/pkg/util/mocks"
-	mockconditions "github.com/openshift/gcp-project-operator/pkg/util/mocks/condition"
-	mockGCP "github.com/openshift/gcp-project-operator/pkg/util/mocks/gcpclient"
-	testStructs "github.com/openshift/gcp-project-operator/pkg/util/mocks/structs"
 	"google.golang.org/api/cloudresourcemanager/v1"
 	"google.golang.org/api/iam/v1"
 	corev1 "k8s.io/api/core/v1"
-	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+
+	api "github.com/openshift/gcp-project-operator/pkg/apis/gcp/v1alpha1"
+	gcpv1alpha1 "github.com/openshift/gcp-project-operator/pkg/apis/gcp/v1alpha1"
+	. "github.com/openshift/gcp-project-operator/pkg/controller/projectreference"
+	"github.com/openshift/gcp-project-operator/pkg/util"
+	"github.com/openshift/gcp-project-operator/pkg/util/mocks"
+	mockconditions "github.com/openshift/gcp-project-operator/pkg/util/mocks/condition"
+	mockGCP "github.com/openshift/gcp-project-operator/pkg/util/mocks/gcpclient"
+	testStructs "github.com/openshift/gcp-project-operator/pkg/util/mocks/structs"
+)
+
+var (
+	fakeError            = errors.New("fakeError")
+	stopProcessingResult = util.OperationResult{
+		RequeueDelay:   0,
+		RequeueRequest: false,
+		CancelRequest:  true,
+	}
```

Comment:
should't we be able to use `util.StopProcessing()` directly?

Reply:
Done.